Base/Logger/Logger.Static/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set_target_properties(${TARGET} PROPERTIES
 target_link_libraries(${TARGET} PUBLIC
         Base.Logger.Public
         Utility.Public
+        Network.Public
```

Comment:
Why do you connect network to loggere??

Reply:
For SafeQueue

Reply:
It will be a new task, am I right?

Reply:
You could push this PR. But please, make new Issue for moving SafeQueue to Utils :)